lib/log_ratelimit.c

```diff
@@ -74,6 +82,9 @@ log_ratelimit_allow(struct log_ratelimit_state *lrs)
 {
 	uint64_t now;
 
+	if (unlikely(!enabled))
+		return true;
+
 	if (lrs->interval_cycles == 0)
 		return true;
```

Comment:
Add `unlikely()` to the `if` test above and the following comment before:
`/* unlikely() reason: all log is rate limited in production. */`

I cannot comment on line `if (lrs->end < now)` below, so assume that the following text applies to it:

Add `unlikely()` to the `if` test above and the following comment before:
`/* unlikely() reason: there is only one reset every @lrs->interval_cycles. */`